src/services/entity.js

```diff
@@ -204,4 +204,65 @@ module.exports = class EntityHelper {
 			throw error
 		}
 	}
+
+	/**
+	 * Get details of entity
+	 * @method
+	 * @name details
+	 * @param {Object} query - query params
+	 * @param {String} userId - logged in user id.
+	 * @param {String} searchText - search label in entity.
+	 * @param {Integer} page -  page no.
+	 * @param {Integer} pageSize -  page limit per api.
+	 * @returns {JSON} - Entity search matched response.
+	 */
+	static async details(query, userId, searchText, pageNo, pageSize) {
+		try {
+			let entityType = query.entity_type_id ? query.entity_type_id : ''
+			let filter
+			if (query.read_user_entity == true) {
```

Comment:
What does the key read_user_entity mean? When will it be true and false?

Reply:
read_user_entity = = true, it reads requested user created entity and entity created by id 0, 
which means default entities,
The same logic is there in existing read api also